projects/components/src/table/header/table-header-cell-renderer.component.ts

```diff
@@ -155,6 +190,19 @@ export class TableHeaderCellRendererComponent implements OnInit, OnChanges {
     this.classes = this.buildClasses();
   }
 
+  public onToggleAllSelectedChange(allSelected: boolean): void {
+    this.allRowsSelected = allSelected;
+    this.allRowsSelectionChange.emit(allSelected);
+  }
+
+  public getHeaderCheckboxTooltip(): string {
+    return this.indeterminateRowsSelected
+      ? 'Some rows are selected'
+      : this.allRowsSelected
+      ? 'All rows in the current page are selected'
```

Comment:
Two of these tooltips say what is currently the state. However, the final tooltip says what interacting with the control will do. Should we change the last one to 'None of the rows in the current page are selected' to align with the others? Or, should we change the others to state what action will happen if clicked?

Reply:
Yeah, I added that because when the tooltip is not selected and the user hovers over it for the first time, they will get to know what the selection would do. I am fine with updating the text like you suggested.